include/vcpkg-test/util.h

```diff
@@ -82,6 +82,21 @@ namespace vcpkg
     {
         return os << "LL" << std::quoted(value.data());
     }
+
+    inline std::ostream& operator<<(std::ostream& os, const Path& value) { return os << value.native(); }
+
+    template<class T, class = decltype(std::declval<std::ostream&>() << std::declval<const T&>())>
```

Comment:
I'm pretty sure this isn't quite correct when T is a reference type. It's test code so I'm not too worried about that though....

Reply:
If you use trailing return type you can avoid the decltype stuff I think:

```suggestion
    template<class T>
    inline auto operator<<(std::ostream& os, const Optional<T>& value)
        -> decltype(os << *(value.get()))
```